source/sass/buyers-guide/bg-main.scss

```diff
@@ -95,14 +94,3 @@ img.thumb-border {
 @import "./views/product";
 @import "./views/about";
 @import "./views/category";
-
-// Force Buyer's Guide links to use the old blues until
-// we tackle this cleanup ticket https://github.com/mozilla/foundation.mozilla.org/issues/2665
-
-a {
```

Comment:
no need to have these overrides as we want the links to follow default foundation site link's behaviour